### PR TITLE
ZK-4390: Embed zul pages by script tag

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zktest/src/archive/WEB-INF/web.xml [hunk 1]
```diff
@@ -274,4 +274,13 @@ Copyright (C) 2004 Potix Corporation. All Rights Reserved.
 		<exception-type>org.zkoss.zktest.test2.B86_ZK_3679Exception</exception-type>
 		<location>/test2/B86-ZK-3679-errhandler.zul</location>
 	</error-page>
+	<!-- ZK-4390 -->
```
Comment (DevChu) [inline zktest/src/archive/WEB-INF/web.xml]: auto

File: zktest/src/archive/test2/F90-ZK-4390test.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	    <title>Title</title>
+	</head>
+	<body>
+		<button id="btn">Click it, and you should see "Foo0 Bar0" messages twice in the zk log area.</button>
+		<div id="aaa">
+		    Loading...
+		</div>
+		<script defer src="http://localhost:8080/zktest/test2/F90-ZK-4390.zul?zk.domId=aaa"></script>		
+		<script defer>
+			window.addEventListener('load', function(){
```
Comment (DevChu) [inline zktest/src/archive/test2/F90-ZK-4390test.html]: indent

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**